rui/app/locales/en/translations.js

```diff
@@ -97,42 +97,41 @@ export default {
     title: "Settings",
     subtitle: "Manage the configuration of ROSE.",
     language: "Language",
-    languageLabel: "Choose your preferred language, or use the default language from the browser (“auto detect” option).",
+    languageLabel: "Choose your preferred language, or use the browser default language (\"auto detect\" option).",
```

Comment:
Why not the correct quotation marks: https://en.wikipedia.org/wiki/Quotation_marks_in_English ?
Typography ❤️